microsoftgraph/microsoft-graph-explorer-v4
# Fix: Narrator announces Tab items activation

## Overview

This PR adds aria-controls to our pivot items (tab) to enable the screen readers to announce when a tab has been selected and that it contains information 

Fixes #1148 

## Testing Instructions

* Load GE
* Turn on the Narrator 'Ctrl+Win+Enter' keys
* Navigate to 'Request Body' tab key and right arrow key and select it by enter/space key 
* Repeat on other Tabs i.e Request Headers, Modify Permissions... etc
* Observe the Narrator's narration
